web-server/.terraform/plugins/linux_amd64/lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file shouldn't be in Git - it changes every time _Terraform_ is used. It's just a lock.